.docker/docker_start_script.sh

```diff
@@ -1,28 +1,7 @@
 #!/usr/bin/env bash
-#Configure web server
 
-# cd /usr/src/app/e-mission-server
-
-#set database URL using environment variable
-echo ${DB_HOST}
-if [ -z ${DB_HOST} ] ; then
-    local_host=`hostname -i`
-    jq --arg db_host "$local_host" '.timeseries.url = $db_host' conf/storage/db.conf.sample > conf/storage/db.conf
-else
-    jq --arg db_host "$DB_HOST" '.timeseries.url = $db_host' conf/storage/db.conf.sample > conf/storage/db.conf
-fi
 cat conf/storage/db.conf
 
```

Comment:
If we have removed this, there is really no reason to `cat conf/storage/db.conf` since it won't exist anyway